src/ui_basic/dropdown.h

```diff
@@ -316,12 +399,42 @@ template <typename Entry> class Dropdown : public BaseDropdown {
 	/// Removes all elements from the list.
 	void clear() {
 		BaseDropdown::clear();
-		entry_cache_.clear();
+		filtered_entries.clear();
+		unfiltered_entries.clear();
 	}
 
 private:
-	// Contains the actual elements. The BaseDropdown registers the indices only.
-	std::deque<std::unique_ptr<Entry>> entry_cache_;
+	class ExtendedEntry {
+	public:
+		const std::string name;
+		const Entry value;
+		const Image* img;
```

Comment:
```suggestion
		const Image* img = nullptr;
```

Reply:
Turns out my suggestion here makes the compiler unhappy…
Please revert this change, sorry :(